test/extract_method_baseline/filtering/test_filtering.py

```diff
@@ -0,0 +1,79 @@
+from typing import List
+from pathlib import Path
+from unittest import TestCase
+
+from veniq.baselines.semi.filter import filter_extraction_opportunities, ExtractionOpportunity
+from veniq.ast_framework import AST, ASTNode, ASTNodeType
+from veniq.utils.ast_builder import build_ast
+
+
+class FilteringTestCase(TestCase):
+    def test_non_continuos_statements_range(self):
+        # lines 9 and 15
+        self._opportunity_test_helper([59, 71], False)
```

Comment:
I did not catch how the test works. I first thought that `[59, 71]` is extraction lines range, but it's not

Reply:
The `ExtractionOpportunity` is a list of ASTNode. Here is ASTNode IDs. They get transformed into ASTNodes in `_create_extraction_opportunity` method. There is no other way to point directly to specific node. Comments in each test help to find specified nodes in source code.

Reply:
I recommend redo the test interface and pass line ranges. It's more intuitive and does not depend on ID assignment which we do not control here.

Reply:
Fixed